openpype/hosts/fusion/plugins/publish/validate_saver_resolution.py

```diff
@@ -87,19 +38,80 @@ def process(self, instance):
 
     @classmethod
     def get_invalid(cls, instance):
-        resolution = cls.get_resolution(instance)
+        saver = instance.data["tool"]
+        try:
+            resolution = cls.get_resolution(instance)
+        except PublishValidationError:
+            resolution = None
```

Comment:
Sorry, what is this fixing? This feels like a very odd fix - capturing the actual error, etc.

This doesn't look like the correct way of fixing this?

Reply:
Another option would be to put there `return [saver]`.

`get_invalid` method is not fixing anything, it is only used to select/highlight broken item in a workfile if `Show me invalid` button is pressed in Publisher UI.

Reply:
I see - fine with me. Very used to having `get_invalid` also used in `process` which made me confused I think. Thanks.